CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+### Contribution guide
```

Comment:
I am not sure that is helpful for contributors. I mean you should add a subheader about code style checking for example and a todo about contribution information. I will fill it in later

Reply:
I am afrad that a lot of people do not know how to run static analysis tasks in gradle. And my implementation with buildsrc looks beautiful, but not very known for amature developers. 

I beat that after a lot of developers will try to contribute to you, they would have a problem with that :)

Reply:
Wow, I did not know it. I thought it is obvious and everyone can run analyzers with Gradle. Then thank you!

Initially, we did not add analyzers because we had a lot of different attempts and approaches to make Reflekt and it was lost between them.